crates/sui-package-resolver/Cargo.toml

```diff
@@ -13,6 +13,7 @@ async-trait.workspace = true
 bcs.workspace = true
 move-binary-format.workspace = true
 move-core-types.workspace = true
+move-command-line-common.workspace = true
```

Comment:
Can you leave a TODO here about what this is used for and how we should factor that out?